File: Phase01-02/src/main/java/TimeRange.java

// Parameter Object to avoid Long Parameter List.
public class TimeRange {
    private final long start;
    private final long end;

    public TimeRange(long start, long end) {
        this.start = start;
        this.end = end;
    }

    @Override
    public String toString() {
        return "(" + calculateTimeTaken() + " seconds)";
    }

    private double calculateTimeTaken() {
        return (end - start) / 1e9;
    }
}

File: Phase05/InvertedIndex/TimeRange.cs
namespace InvertedIndex;

public class TimeRange
{
    private readonly long _start;
    private readonly long _end;

    public TimeRange(long start, long end)
    {
        this._start = start;
        this._end = end;
    }

    public override string ToString()
    {
        return "(" + CalculateTimeTaken() + " seconds)";
    }

    private double CalculateTimeTaken()
    {
        return (_end - _start) / 1e9;
    }
}